els-front/src/components/UpdatePassword.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Form, Field } from 'react-final-form'
+import { useNavigate } from "react-router-dom";
+import api from '../plugins/axios'
+import { successNotify } from "../plugins/toast";
+
+const UpdatePassword = () => {
+    const navigate = useNavigate();
+
+    const handleUpdate = async(formValues) => {
+        const response = await api.post(`user/update-password`, formValues)
+
+        if(response) {
+            successNotify('Updated password successfully')
+            navigate(0)
+        }
+        
+    }
+
+    const onSubmit = (formValues) => {
+        handleUpdate(formValues)
+    } 
+
+    const validateForm = (formValues) => {
+        const errors = {};
+
+        if(!formValues.password) 
+            errors.password = "Password is required"
+        
+        if(!formValues.new_password)
+            errors.new_password = "New Password is required"
+        else if(formValues.new_password !== formValues.new_password_confirmation){
+            errors.new_password = "New Password does not match"
+            errors.new_password_confirmation = "New Password does not match"
+        }
+
+        return errors;
+    } 
+
+    return (
+        <div className="w-full h-full mt-16 ml-2 bg-white border rounded-lg shadow-md sm:p-8 dark:bg-gray-800 dark:border-gray-700">
+            <div className="text-4xl font-medium">Update password</div>
+            <Form 
+                onSubmit={ onSubmit }
```

Comment:
is below not working? so we can delete the onSubmit function
```suggestion
                onSubmit={ handleSubmit }
```